### PR TITLE
Update Node js

### PR DESCRIPTION
**Pull request by MichaelHinrichs**
### SteamDB app page links to a few games using this
https://steamdb.info/app/793490/

### Brief explanation of the change
Find the Node JS libraries for mac OS and Linux.

### CODE REVIEW

File: rules.ini [hunk 1]
```diff
@@ -241,6 +241,8 @@ Lua[] = \.lua$
 Miles_Sound_System = (?:^|/)mss(?:16|32|64)\.dll$
 Modio = (?:^|/)mod\.?io
 NodeJS = (?:^|/)node\.dll$
+NodeJS[] = (?:^|/)node\.dll$
+NodeJS[] = (?:^|/)libnode\.(?:dylib|so)$
```
Comment (Lifeismana) [inline rules.ini:245]:
```suggestion
NodeJS = (?:^|/)(?:lib)?node\.(?:dll|dylib)$
```
unless you have an example of it's usage only on linux, let's not add it

**Comment by Lifeismana** [comment]
also, remove the unrelated commits from your pr

**Comment by MichaelHinrichs** [comment]
> also, remove the unrelated commits from your pr

How?

**Comment by SteamTracker** [comment]
After running changed rules in this pull request *(this bot only tests rules.ini changes)*:

Changed technologies: **SDK.NodeJS**

**5354** apps still match.

These apps now match: **(26)** [239820](https://steamdb.info/app/239820/depots/) [335240](https://steamdb.info/app/335240/depots/) [370900](https://steamdb.info/app/370900/depots/) [404700](https://steamdb.info/app/404700/depots/) [416000](https://steamdb.info/app/416000/depots/) [544290](https://steamdb.info/app/544290/depots/) [552540](https://steamdb.info/app/552540/depots/) [603750](https://steamdb.info/app/603750/depots/) [645690](https://steamdb.info/app/645690/depots/) [658990](https://steamdb.info/app/658990/depots/) [738130](https://steamdb.info/app/738130/depots/) [740790](https://steamdb.info/app/740790/depots/) [793490](https://steamdb.info/app/793490/depots/) [845990](https://steamdb.info/app/845990/depots/) [1008020](https://steamdb.info/app/1008020/depots/) [1161330](https://steamdb.info/app/1161330/depots/) [1184330](https://steamdb.info/app/1184330/depots/) [1185340](https://steamdb.info/app/1185340/depots/) [1803530](https://steamdb.info/app/1803530/depots/) [1823490](https://steamdb.info/app/1823490/depots/) [1884070](https://steamdb.info/app/1884070/depots/) [2177610](https://steamdb.info/app/2177610/depots/) [2211180](https://steamdb.info/app/2211180/depots/) [2302100](https://steamdb.info/app/2302100/depots/) [2491950](https://steamdb.info/app/2491950/depots/) [3011320](https://steamdb.info/app/3011320/depots/)

Please confirm that all these changed apps are correct.

**Comment by Lifeismana** [comment]
merged in c4fa73f4a55a74d538b2a8fbaf905ee2989c85d1

**Pull request closed without merging**